src/main/java/org/jmhsrobotics/frc2024/RobotContainer.java

```diff
@@ -283,6 +290,7 @@ public ArmPIDSubsystem getArmSubsystem() {
 	public void configureSmartDashboard() {
 		SmartDashboard.putData("Scheduler", CommandScheduler.getInstance());
 
+		CameraServer.startAutomaticCapture("Kirby", 0);
```

Comment:
You dont need this line. This is only used for cameras plugged into usb on the RIO. the camera streams are already published to smart dashboard

Reply:
👌